components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/JsGraphBuilder.java

```diff
@@ -572,6 +580,33 @@ public static void addPrompt(String templateId, Map<String, Object> parameters,
         addHandlers(newNode, handlers);
     }
 
+    /**
+     * Loads the required function library from the database.
+     *
+     * @param functionLibraryName functionLibraryName
+     * @return functionLibraryScript
+     * @throws FunctionLibraryManagementException
+     */
+    public String loadLocalLibrary(String functionLibraryName) throws FunctionLibraryManagementException {
+
+        FunctionLibraryManagementService functionLibMgtService = FrameworkServiceComponent.
+                getFunctionLibraryManagementService();
+        FunctionLibrary functionLibrary;
+        String libraryScript = null;
+        boolean isFunctionLibrary = false;
+        isFunctionLibrary = functionLibMgtService.isFunctionLibraryExists(functionLibraryName,
```

Comment:
Shall we use the same functionLibMgtService.getFunctionLibrary() method and handle non existing function library name case if the result was null.